assets/content/cookbook/Data structures/ring-array.md

```diff
@@ -89,7 +88,6 @@ class Ring<T> {
 It's easy to implement `undo/redo` operations.
 
 ```haxe
-#if !js @:generic #end
```

Comment:
Did you remove `@:generic` entirely here on purpose?

Reply:
I just want to make the example look simpler, because it generates some **ugly** code in js. 

It will generate the following code:
```js
var History_$Int = function(len) {
}
var Ring_$Int = function(len) {
}
```